static/app/views/alerts/rules/issue/details/sidebar.tsx

```diff
@@ -80,7 +88,11 @@ function Conditions({rule, teams, memberList}: Props) {
               rule.actions.map((action, idx) => {
                 return (
                   <ConditionsBadge key={idx}>
-                    <TextAction action={action} memberList={memberList} teams={teams} />
+                    <TextAction
+                      action={action}
+                      memberList={memberList ?? []}
```

Comment:
What does this look like when a user action is here? Does it handle things nicely while the request is in flight?

Reply:
added a better fallback to Unknown